packages/ui/Marquee/MarqueeBase.js

```diff
@@ -58,7 +57,7 @@ const MarqueeBase = kind({
 		children: PropTypes.node,
 
 		/**
-		 * Function to capture a reference to the client node
+		 * Called to capture a reference to the client node
```

Comment:
'Called with a reference to the client node'?  'Called when mounting or unmounting with a ...'?  Not sure if those are better.